housing_cooperative_base/demo/demo.xml

```diff
@@ -147,8 +147,8 @@
         </record>
         <record id="demo_lease_2" model="hc.lease">
              <field name="tenant_id" ref="res_partner_cooperator_5_demo"/>
-            <field name="start">2018-09-20</field>
-            <field name="expected_end">2021-09-20</field>
+            <field name="start">2018-09-01</field>
```

Comment:
For demo data, you can use limited python expression to generate dates, for example: 

```xml
        <field name="min_recovery_date" eval="(DateTime.today() + relativedelta(days=1) + relativedelta(hours=8)).strftime('%Y-%m-%d %H:%M')"/>

```

Reply:
Thanks! I'll remember this for next time